src/source/ads.rst

```diff
@@ -395,6 +396,17 @@ per customer. Can either be set when creating an ad or when updating an
 existing ad. However, once set, it can no longer be modified. When fetching an
 existing ad which does not have a **vendorId**, the field is omitted.
 
+.. index:: microTip
+.. _ad_microTip:
+
+microTip
+""""""""
+A short freeform text with a maximum length of 18 characters, excluding any characters in ``.,/@#<>``.
+It is a feature used in the context of SMB bundles (currently only by Marktplaats tenant),
+and will only be usable by enabled sellers.
+It will be shown as a "sticker" (giving extra attention) on the ad in the search results.
```

Comment:
remove the 'sticker' mention, it's there to give extra attention.

Reply:
fixed